test/CMakeLists.txt

```diff
@@ -7,3 +7,9 @@ rock_testsuite(testGUI suite.cpp testWidget.cpp
 rock_testsuite(testCore suite.cpp testTransformerGraph.cpp
     DEPS vizkit3d
     LIBS ${Boost_THREAD_LIBRARY} ${Boost_SYSTEM_LIBRARY})
+
+rock_executable(vizkit3d-bin NOINSTALL
+    SOURCES vizkit3d.cpp
```

Comment:
this is just for testing purposes but not unit tests.. maybe should be removed, maintainer should decide